packages/terra-button/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+* Changed
+  * Changed implementation from Class Component to Functional Component.
```

Comment:
We can remove this line as this information is not relevant for consumers.

Reply:
Done